src/Parallel/MPIBasic.h

```diff
@@ -89,6 +103,13 @@ class MPIBasic
 		return m_size;
 	}
 
+	/**
+	 * @return The rank of this process within this node
```

Comment:
Consider updating:
return the number of ranks within the shared memory sub-communicator

Reply:
done